portfolio/src/main/webapp/script.js

```diff
@@ -26,16 +26,19 @@ function randomDogImage(){
   imageRemover.appendChild(imageElement);
 }
 
-function loadComments() {
-  fetch('/data').then(response => response.json()).then(data => {
-    const commentListElement = document.getElementById('task-list');
-    data.forEach((task) => {
-      commentListElement.appendChild(createCommentElement(task));
-    })
+/** Displays amount of comments chosen by the user. */
+function getNumComments(commentsNum) {
+  fetch('/data?numOfComments=' + commentsNum).then(response => response.json()).then((comments) => {
+    const taskListElement = document.getElementById('task-list');
+    taskListElement.innerHTML="";
+      for (let i=0; i<commentsNum; i++)
```

Comment:
You'll want to iterate up to the length of comments. If comments.length < commentsNum, you'll get an error.